### PR TITLE
Live migration tests - Additions and fixes

### PR DESCRIPTION
**Pull request by enp0s3**
Test 1785 supposed to be w/o guest agent while actually it does have it.
Needed to re-write test 1785 to be w/o guest agent.

Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

**What this PR does / why we need it**:
We need to test a live migration case where a Linux VM has guest agent inside. This PR adds the missing test. To be more specific, this test was already implemented but labeled incorrectly, so the PR fixes the labeling and creates a test that was really missing in the code (test 1785 becomes test 2653, and re-create test 1785)

**Which issue(s) this PR fixes** 
Test IDs 1785, 2653

**Special notes for your reviewer**:
@ILpinto please take a look

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rmohr** [comment]
@enp0s3 can you please run `make && make generate` and commit the difference? Travis said that the code is not in sync: https://travis-ci.org/kubevirt/kubevirt/builds/637451075#L1447

File: tests/migration_test.go [hunk 1]
```diff
@@ -767,8 +768,54 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
...
+				migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
+				migrationUID := runMigrationAndExpectCompletion(migration, migrationWaitTime)
+
+				// check VMI, confirm migration state
+				confirmVMIPostMigration(vmi, migrationUID)
+
+				By("Checking that the migrated VirtualMachineInstance console has expected output")
+				expecter, err = tests.ReLoggedInFedoraExpecter(vmi, 60)
+				defer expecter.Close()
+				Expect(err).ToNot(HaveOccurred())
```
[the remark this answers is not in the document]
Comment (enp0s3) [reply tests/migration_test.go]: Sure. Added in a new commit

**Comment by danielBelenky** [comment]
/lgtm

I retriggered travis

**Pull request closed without merging**